js/controller.js

```diff
@@ -196,8 +196,8 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         }
 
         // ad events
-        if (!Utils.isIPhone()) {
-          //since iPhone is always playing in full screen and not showing our skin, don't need to render skin
+        if (Utils.canRenderSkin()) {
+          //since iPhone < iOS10 is always playing in full screen and not showing our skin, don't need to render skin
```

Comment:
We should probably move this comment to the canRenderSkin function (or have it in both places).